### ISSUE TITLE
Changing channels while watching a channel requires an explicit STOP first

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google-code-export/mythbox#11]
```

This is not a bug really, but a feature request, you could say.

What steps will reproduce the problem?
1. Watch LiveTV
2. Go to the Guide
3. If you now try to start a new channel, it doesn't work. You have to
explicitly STOP playback (STOP button) before playing a new channel.

What is the expected output? What do you see instead?

I would like MythBox to automatically STOP the playback of the old channel,
and starting the new.

a thought: it would be really cool if it checked to see if there was an
idle tuner, and if so, start buffering the new channel WHILE playing the
old. And when the buffering was complete, stop the old and show the new. It
would just be cool, havn't thought of it much. However you would
temporarily lock up another tuner which is a small drawback.

What version of the product are you using? On what operating system?

658 on Linux

Please provide any additional information below.

If any more information is needed, I'll provide it.


```

Original issue reported on code.google.com by `daniel.a...@gmail.com` on 12 Jan 2009 at 6:52

**Comment by GoogleCodeExporter** [comment]
Original comment by `semir.pa...@gmail.com` on 13 Jan 2009 at 5:06
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `semir.pa...@gmail.com` on 7 May 2009 at 3:41
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in svn 1510 for 0.21 backends. Use chan+/- buttons.
0.22 backends are another story...
```

Original comment by `semir.pa...@gmail.com` on 19 Oct 2009 at 6:20
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
I think this needs to be addressed again, I'm experiencing this on mythbuntu 
10.10 and mythbox 1.0.2
```

Original comment by `EvanJame...@gmail.com` on 7 Mar 2011 at 4:27

**Comment by GoogleCodeExporter** [comment]
```
I'm seeing strange channel changing behavior too.  I'm running the mythbackend 
0.23 with fixes on Ubuntu 10.04, a Hauppague 950Q tuner, and mythbox 1.0.4 on 
XBMC.  The up/down arrows on my keyboard seek, and I can't find any other way 
to change a channel without stopping and then reselecting from the channel 
list.  It also seems a bit unstable and freezes quite often.
```

Original comment by `mcew...@gmail.com` on 22 Jun 2011 at 11:01
